src/msg/producer/writer/message_writer.go

```diff
@@ -298,9 +303,10 @@ func (w *messageWriterImpl) nextRetryNanos(writeTimes int, nowNanos int64) int64
 }
 
 func (w *messageWriterImpl) Ack(meta metadata) bool {
-	acked, initNanos := w.acks.ack(meta)
+	// acked, initNanos := w.acks.ack(meta)
+	acked, _ := w.acks.ack(meta)
 	if acked {
-		w.m.messageConsumeLatency.Record(time.Duration(w.nowFn().UnixNano() - initNanos))
+		// w.m.messageConsumeLatency.Record(time.Duration(w.nowFn().UnixNano() - initNanos))
```

Comment:
Hi @robskillington a lot of these metrics are commented out, any reason for that? Can we uncomment them?